<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">GenetiKALK</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/img.ico</normaloff>../img/img.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="kalkTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>450</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabDNA">
     <attribute name="title">
      <string>DNA</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>551</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>ATCGGCATTGCA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>T/U</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>41</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>371</width>
        <height>141</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Testo vario, tipo sdgfgjhvjh&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Lorem ipsum sic dolor amet&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Salva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Svuota</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Carica</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>371</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Operazioni base</string>
      </property>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Direzione/Volume</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Concatena</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Massa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Lunghezza</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>200</y>
        <width>221</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Operazioni speciali</string>
      </property>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>CalcolaComp/Splicing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ScambiaComp/TrovaAmm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Traduci in DNA/DelComp</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Traduci in RNA/Proteina</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>RST</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRNA">
     <attribute name="title">
      <string>RNA</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPro">
     <attribute name="title">
      <string>Proteina</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
